riff-raff/app/controllers/Application.scala

```diff
@@ -84,19 +86,19 @@ class Application(prismLookup: PrismLookup, deploymentTypes: Seq[DeploymentType]
     Ok(views.html.index(request, documentation))
   }
 
-  def deployInfoData = AuthAction { implicit request =>
+  def deployInfoData = authAction { implicit request =>
```

Comment:
I don't feel strongly about it, but followed the convention I saw in the docs here: https://www.playframework.com/documentation/2.6.x/ScalaActionsComposition